grazel-gradle-plugin/src/main/kotlin/com/grab/grazel/extension/MavenInstallExtension.kt

```diff
@@ -66,6 +66,7 @@ data class MavenInstallExtension(
     var jetifyIncludeList: ListProperty<String> = objects.listProperty(),
     var jetifyExcludeList: ListProperty<String> = objects.listProperty(),
     var versionConflictPolicy: String? = null,
+    var credentials: Boolean = true,
```

Comment:
This is not as descriptive about what this flag does, maybe does `includeCredentials` reads better?

Reply:
Make sense. Have changed it to `includeCredentials` in new commit.